### PR TITLE
Add workflows for triggering hosting request checking

### PR DESCRIPTION
**Pull request by timja**
Triggers on:
- Issue open
- Issue edit
- Issue comment with `/hosting re-check`
- Issue edit with `/hosting re-check`

See testing: https://github.com/timja/repository-permissions-updater/issues/1

e.g. https://github.com/timja/repository-permissions-updater/issues/1#issuecomment-1003568211

### CODE REVIEW

File: .github/workflows/hosting-comment-checker.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          java-version: '11'
+          cache: 'maven'
+      - name: Build with Maven
+        run: mvn -B package -Dspotbugs.skip
+      - name: Run hosting checker
+        run: |
+          java -cp target/repository-permissions-updater-1.0-SNAPSHOT-bin/repository-permissions-updater-1.0-SNAPSHOT.jar io.jenkins.infra.repository_permissions_updater.hosting.HostingChecker ${{ github.event.issue.number }}
+        env:
+          GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}
+          ORG_NAME: timja
```
Comment (timja) [inline .github/workflows/hosting-comment-checker.yml:40]:
```suggestion
```

File: .github/workflows/hosting-issue-checker.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          java-version: '11'
+          cache: 'maven'
+      - name: Build with Maven
+        run: mvn -B package -Dspotbugs.skip
+      - name: Run hosting checker
+        run: |
+          java -cp target/repository-permissions-updater-1.0-SNAPSHOT-bin/repository-permissions-updater-1.0-SNAPSHOT.jar io.jenkins.infra.repository_permissions_updater.hosting.HostingChecker ${{ github.event.issue.number }}
+        env:
+          GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}
+          ORG_NAME: timja
```
Comment (timja) [inline .github/workflows/hosting-issue-checker.yml:29]:
```suggestion
```

File: .github/workflows/hosting-issue-checker.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          distribution: 'temurin' # See 'Supported distributions' for available options
+          java-version: '11'
+          cache: 'maven'
+      - name: Build with Maven
+        run: mvn -B package -Dspotbugs.skip
+      - name: Run hosting checker
+        run: |
+          java -cp target/repository-permissions-updater-1.0-SNAPSHOT-bin/repository-permissions-updater-1.0-SNAPSHOT.jar io.jenkins.infra.repository_permissions_updater.hosting.HostingChecker ${{ github.event.issue.number }}
+        env:
+          GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}
```
Comment (timja) [inline .github/workflows/hosting-issue-checker.yml:28]:
```suggestion
          GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}

```

File: .github/workflows/hosting-comment-checker.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+          distribution: 'temurin' # See 'Supported distributions' for available options
+          java-version: '11'
+          cache: 'maven'
+      - name: Build with Maven
+        run: mvn -B package -Dspotbugs.skip
+      - name: Run hosting checker
+        run: |
+          java -cp target/repository-permissions-updater-1.0-SNAPSHOT-bin/repository-permissions-updater-1.0-SNAPSHOT.jar io.jenkins.infra.repository_permissions_updater.hosting.HostingChecker ${{ github.event.issue.number }}
+        env:
+          GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}
```
Comment (timja) [inline .github/workflows/hosting-comment-checker.yml:39]:
```suggestion
          GITHUB_OAUTH: ${{ secrets.GITHUB_TOKEN }}

```

File: .github/workflows/hosting-comment-checker.yml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin' # See 'Supported distributions' for available options
+          java-version: '11'
+          cache: 'maven'
+      - name: Build with Maven
+        run: mvn -B package -Dspotbugs.skip
+      - name: Run hosting checker
+        run: |
+          java -cp target/repository-permissions-updater-1.0-SNAPSHOT-bin/repository-permissions-updater-1.0-SNAPSHOT.jar io.jenkins.infra.repository_permissions_updater.hosting.HostingChecker ${{ github.event.issue.number }}
```
Comment (halkeye) [inline .github/workflows/hosting-comment-checker.yml:37]: not that I think version number will ever change, but you could use * instead of 1.0 right?

**Comment by halkeye** [comment]
I know i prefer jenkins instead of github actions, but seems like it'll work, so lgtm

**Review by halkeye — APPROVED** [review]

File: .github/workflows/hosting-comment-checker.yml (hunk 5 above)
Comment (timja) [reply .github/workflows/hosting-comment-checker.yml:37]: Probably but it’ll never change

**Comment by timja** [comment]
> I know i prefer jenkins instead of github actions, but seems like it'll work, so lgtm

I think it would be a lot more difficult to do something like this with Jenkins, I’m sure it’s possible but I think this is a better fit

**Pull request merged**